src/common/product_models/heatmap_common_models.py

```diff
@@ -99,6 +99,20 @@ class ColumnInformation(BaseModel):
     )
 
 
+def transfer_col_heatmap_to_attribs(col: ColumnInformation) -> attribs_models.AttributesColumn:
```

Comment:
Are you going to need to use this for the display name column, for which you'll also need to transfer the filter strategy?

Reply:
This is only for cols exist in heatmap meta. Won't be able to use this for display_name, kbase_id, etc cols, they are not in heatmap meta. I am planning to manually add them.

Reply:
:+1: